### ISSUE TITLE
Option to use system font

### ISSUE THREAD

**Issue by etenium** [ImranR98/Obtainium#415]
The custom font used by the app right now looks very much out of place. Could you add a toggle in the settings to simply use the system font?

**Comment by ImranR98** [comment]
Blocked by https://github.com/flutter/flutter/issues/48381

**Comment by re7gog** [comment]
![Screenshot_2023-12-31-19-24-19-907_dev imranr obtainium](https://github.com/ImranR98/Obtainium/assets/114265068/4efc0032-158c-4268-b9bb-1a2cecae78f4)
Yey

**Comment by 0-BlackSpectrum-0** [comment]
> ![Screenshot_2023-12-31-19-24-19-907_dev imranr obtainium](https://github.com/ImranR98/Obtainium/assets/114265068/4efc0032-158c-4268-b9bb-1a2cecae78f4)
> Yey

Can't you add it? please,  I want it to use sys font as well, like this it's more consistent to have everything use same Font!

**Comment by ImranR98** [comment]
Forgot to close this issue. @re7gog already added it.

**Comment by 0-BlackSpectrum-0** [comment]
> Forgot to close this issue. @re7gog already added it.

My bad I didn't saw that sys font option in settings,  thanks but when I turned it on it doesn't use system font here's the screenshots:

System font (chrome):

![Screenshot_20240302-161414_Chrome Canary.png](https://github.com/ImranR98/Obtainium/assets/79910450/524d3967-be18-4b8f-818c-adecdc24b803)

Obtainium sys font enabled:

![Screenshot_20240302-161116.png](https://github.com/ImranR98/Obtainium/assets/79910450/888a5f33-2849-4654-b598-554ea0c86944)

Obtainium Home screen (doesn't use system font):

![Screenshot_20240302-161122.png](https://github.com/ImranR98/Obtainium/assets/79910450/3b94961e-b355-4382-ada2-f397907c0ddf)

**Comment by ImranR98** [comment]
Guess the implementation only works for some OSes/fonts. Might have to wait for proper Flutter support (linked issue above).

**Comment by 0-BlackSpectrum-0** [comment]
I saw a app that had an option to import a font zip file and use it, you can implement that if possible! 
(App I'm talking about is glassdown, but it was taken down)

**Comment by rafnov** [comment]
TL;Dr
Very much needed. Font now is so stretched. System  be default.
